config/settings.yml

```diff
@@ -26,6 +26,7 @@ hesa:
   current_collection_start_date: "2022-08-01"
   username: <get from secrets>
   password: <get from secrets>
+  upload_id: null
```

Comment:
I don't think you need this `Settings.hesa.upload_id` will return nil by default if it's not defined. And also, `null` ?!